main.py

```diff
@@ -15,28 +16,89 @@ class Robot:
     UPPER_POSITION = 30
     LOWER_POSITION = 0
     WIGGLE_AMOUNT = 5
+    INACTIVITY_PERIOD_S = 15
 
     async def connect(self):
         opts = RobotClient.Options(
             refresh_interval=0,
             dial_options=DialOptions(credentials=secrets.creds)
         )
         self.robot = await RobotClient.at_address(secrets.address, opts)
-        self.servo = Servo.from_robot(self.robot, "servo")
+        self._servo = Servo.from_robot(self.robot, "servo")
+
+        self._mutex = threading.Lock()
+        self._count = 0  # Number of hands currently raised
+
+        # We have a background thread that wiggles the hand when it's been
+        # raised for a long time. This condition variable is how to shut that
+        # down.
+        # It would be intuitive to pass self._mutex to threading.Condition, but
+        # that will result in deadlock when you have the mutex and wait for the
+        # thread to shut down.
```

Comment:
Not sure if this comment is useful: it was an insight I missed at first and had to debug, but perhaps I'm just rusty on my condition variables and this is obvious to folks who use them regularly.